app/services/LocationService.js

```diff
@@ -24,114 +23,7 @@ export class LocationData {
   }
 
   async getLocationData() {
-    const locationArrayString = await GetStoreData(LOCATION_DATA);
-    let locationArray = [];
-
-    if (locationArrayString !== null) {
-      locationArray = JSON.parse(locationArrayString);
-    }
-
-    return locationArray;
-  }
-
-  async getPointStats() {
-    const locationData = await this.getLocationData();
-
-    let lastPoint = null;
-    let firstPoint = null;
-    let pointCount = 0;
-
-    if (locationData.length) {
-      lastPoint = locationData.slice(-1)[0];
-      firstPoint = locationData[0];
-      pointCount = locationData.length;
-    }
-
-    return {
-      lastPoint,
-      firstPoint,
-      pointCount,
-    };
-  }
-
-  saveLocation(location) {
-    // Persist this location data in our local storage of time/lat/lon values
-    this.getLocationData().then(locationArray => {
-      // Always work in UTC, not the local time in the locationData
-      let unixtimeUTC = Math.floor(location['time']);
-      let unixtimeUTC_28daysAgo = unixtimeUTC - 60 * 60 * 24 * 1000 * 28;
-
-      // Verify that at least the minimum amount of time between saves has passed
-      // This ensures that no matter how fast GPS coords are delivered, saving
-      // does not happen any faster than the minLocationSaveInterval
-      if (locationArray.length >= 1) {
-        let lastSaveTime = locationArray[locationArray.length - 1]['time'];
-        if (lastSaveTime + this.minLocationSaveInterval > unixtimeUTC) {
-          //console.log('[INFO] Discarding point (too soon):', unixtimeUTC);
-          return;
-        }
-      }
-
-      // Curate the list of points, only keep the last 28 days
-      let curated = [];
-      for (let i = 0; i < locationArray.length; i++) {
-        if (locationArray[i]['time'] > unixtimeUTC_28daysAgo) {
-          curated.push(locationArray[i]);
-        }
-      }
-
-      // Backfill the stationary points, if available
```

Comment:
Sorry I'm coming late to the party here ... it looks like we no longer have this backfill capability anymore.  Is that intended?

I think without this backfill we lose the capability to intersect in cases where the phone was essentially still for a long period but an intersecting time-location is reported that matches.  It also fills in large gaps of non-movement, which is helpful when the data is loaded by a health authority.

Reply:
See below comment.